demuxalot/BDRhapsody_specific.py

```diff
@@ -0,0 +1,36 @@
+"""
+This file defines callbacks that are optimized for cellranger
```

Comment:
pls adjust description to rhapsody 🙂 

Also, if you can write comment about BAM tags used by rhapsody (or maybe there is a link for this?), this also would be helpful.

Also worth addition:  version of rhapsody software you tested this with, in case something changes in their format.